.clang_complete

```diff
@@ -1,4 +1,7 @@
 -I/home/igor/.config/sublime-text-3/Packages/EasyClangComplete/src blah
 -I/home/igor/.config/sublime-text-3/Packages/EasyClangComplete
 -Ilocal_folder
--std=c++11
+-IC:\GNUstep\GNUstep\System\Library\Headers
```

Comment:
I couldn't find documentation for anything other than GNUstep for Windows/Linux objc/c++ development. Good idea, this makes sense to put in plugin settings for the common case to work easily.
https://github.com/niosus/EasyClangComplete/pull/219/commits/ebaeffc5f95b7d0f72ed37d846921bb8271098f0